src/create-new-game.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable import/prefer-default-export */
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+
+async function getGameID() {
```

Comment:
- [ ] Your `getGameID` does not follow the  `arrow function` syntax. Kindly ensure to change all methods having the keyword function to `arrow functions` so as to enhance cleaner and simpler syntax of method definition

Reply:
Thanks for the tip, @rloterh! :pray: